ui/component/walletTipAmountSelector/view.jsx

```diff
@@ -34,25 +34,32 @@ type Props = {
   uri: string,
   onTipErrorChange: (string) => void,
   activeTab: string,
+  shouldDisableReviewButton: (boolean) => void
 };
 
 function WalletTipAmountSelector(props: Props) {
-  const { balance, amount, onChange, activeTab, claim, onTipErrorChange } = props;
+  const { balance, amount, onChange, activeTab, claim, onTipErrorChange, shouldDisableReviewButton } = props;
   const [useCustomTip, setUseCustomTip] = usePersistedState('comment-support:useCustomTip', false);
   const [tipError, setTipError] = React.useState();
 
   const [canReceiveFiatTip, setCanReceiveFiatTip] = React.useState(); // dont persist because it needs to be calc'd per creator
   const [hasCardSaved, setHasSavedCard] = usePersistedState('comment-support:hasCardSaved', false);
 
+  // if it's fiat but there's no card saved OR the creator can't receive fiat tips
+  const shouldDisableFiatSelectors = (activeTab === TAB_FIAT && (!hasCardSaved || !canReceiveFiatTip));
+
+  /**
+   * whether tip amount selection/review functionality should be disabled
+   * @param [amount] LBC amount (optional)
+   * @returns {boolean}
+   */
   function shouldDisableAmountSelector(amount) {
-    return (
-      (amount > balance && activeTab !== TAB_FIAT) || (activeTab === TAB_FIAT && (!hasCardSaved || !canReceiveFiatTip))
-    );
+    // if it's LBC but the balance isn't enough, or fiat conditions met
+    // $FlowFixMe
+    return (amount > balance && activeTab !== TAB_FIAT) || shouldDisableFiatSelectors;
```

Comment:
if you supply an inititial value to useState(false), flow wouldn't complain here.